app/templates/sponsors.hbs

```diff
@@ -60,7 +60,15 @@
     </ul>
   </section>
 
-  <p class="text-center my-5">
-    A special thanks to <a href="https://dribbble.com/mg" rel="nofollow noopener noreferrer" target="_blank">Matt Grantham</a> for design of the original Ember website.
-  </p>
+  <section class="my-5" aria-labelledby="ember-sponsors-and-friends-individual-sponsors">
+    <h2 id="ember-sponsors-and-friends-individual-sponsors">Individual Sponsors</h2>
+
+    <p>
+      If you can help fund the development of Ember, we encourage you to visit our <a href="https://opencollective.com/emberjs">Open Collective</a> and <a href="https://github.com/sponsors/emberjs">GitHub Sponsors</a> channels. You can provide your donation in either channel. All financial contributions are moved to Open Collective and centrally, publicly managed there.
```

Comment:
I borrowed some words from the [issue description](https://github.com/ember-learn/ember-website/issues/755#issue-802514394) and [GitHub Sponsors page](https://github.com/sponsors/emberjs).